rust/codelist-rs/src/codelist.rs

```diff
@@ -190,6 +190,82 @@ impl CodeList {
     pub fn metadata(&self) -> &Metadata {
         &self.metadata
     }
+
+    /// Truncate codelist entries to 3 digits
+    ///
+    /// # Arguments
+    /// * `term_management` - How to handle ambiguous terms
+    ///
+    /// # Errors
+    /// * `CodeListError::CodeListNotTruncatable` - If the codelist is not ICD10
+    pub fn truncate_to_3_digits(
+        &mut self,
+        term_management: TermManagement,
+    ) -> Result<(), CodeListError> {
+        if !self.codelist_type.is_truncatable() {
+            return Err(CodeListError::CodeListNotTruncatable);
```

Comment:
Can we add to the error to say what Codelist Type was passed through:

```
SNOMED is not truncatable to 3 digits
```

Would be helpful.

Reply:
Yes, that's a good idea.

Reply:
I made that change and committed it. I see it up above in the Commits tab, but I don't see the commit here.

Reply:
Oh, I see...it's down below after these conversations.